makefile

```diff
@@ -378,6 +378,18 @@ phony += clean
 gqlgen:
 	go run github.com/99designs/gqlgen generate
 
+swaggo-install:
+	go install github.com/swaggo/swag/cmd/swag@latest
+
+swaggo-format:
+	swag fmt -g service/service.go
+
+swaggo-build:
+	swag init -g service/service.go
+
+swaggo-render:
```

Comment:
Since it seems like these recipes are not independent of each other and is basically just a list of steps, can we just combine all of these recipes into one target to show the REST v2 docs?